tools_iuc.yaml

```diff
@@ -2036,4 +2036,4 @@ tools:
 
   - name: heinz
     owner: iuc
-    tool_panel_section_label: heinz 
+    tool_panel_section_label: Heinz 
```

Comment:
Can we choose on of the existing categories?

Reply:
After installing "heinz," 4 tools are installed, which form a workflow (together with the tool 'DESeq2' or 'limma' or something similar); I tend to think that these four tools stay together so that users can access them quickly.

![image](https://user-images.githubusercontent.com/11710225/44260745-b0dbae00-a215-11e8-8acb-d90831dc7d55.png)

After going through all the existing categories in https://usegalaxy.eu/, maybe we can put it into "RNA Analysis," but each of the four tools is not so independent or correlated to RNA (though as a whole, it is).

So maybe it is better to make these 4 tools into one folder (or label) called "Heinz"? (I am flexible with this).

Reply:
I do understand, but we already have so many categories and would like to keep them as small as possible. Also I don't think that the word `Heinz` is giving anyone an impression what that is supposed to do. With `epigenetics` or `transcriptomics` people can at least say, yes this is interesting for me or not.

Reply:
No problem, let's put it into ''RNA Analysis". Would be nice if the tool panel can support subfolders (just a naive thought of me) in future. I will change the code now.